src/signal/_proto/gen_proto.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+echo "Compiling proto file(s)..."
+
+protoc sfu.proto \
+	--proto_path=. \
+    --plugin=protoc-gen-ts=../../../node_modules/.bin/protoc-gen-ts \
+    --js_out=import_style=commonjs,binary:./library \
+    --ts_out=service=true:./library
```

Comment:
we should use a docker here. otherwise it depends on the versions installed on the host machine. we have something similar setup in `ion-sfu`

Reply:
fixed https://github.com/pion/ion-sdk-js/commit/b85f295f5516f9c419121229274babc444bb99da